layouts/community/65_ansi_blocker/mechmerlin/readme.md

```diff
@@ -0,0 +1,11 @@
+# MechMerlin's 65_ansi_blocker layout
+
+This is the 65% layout used by u/merlin36, host of the [MechMerlin](www.youtube.com/mechmerlin) 
+YouTube channel.
+
+It is used on his   
+[RGB Doro67](https://github.com/qmk/qmk_firmware/tree/master/keyboards/doro67/rgb)     
+[KBD67 mkii](https://github.com/qmk/qmk_firmware/tree/master/keyboards/kbdfans/kbd67mkiirgb)  
```

Comment:
Just merged 6799 so this path needs updating. :)

```suggestion
[KBD67 mkii](https://github.com/qmk/qmk_firmware/tree/master/keyboards/kbdfans/kbd67/mkiirgb)  
```

Reply:
Done!